src/components/standup/standupContainer.module.scss

```diff
@@ -0,0 +1,157 @@
+$theme-CornFlowerBlue: #1c64f2;
+$theme-Biscay: #1c325e;
+$theme-white: #ffffff;
+$theme-FrenchRose: #f1416c;
+$theme-FunBlue: #1e429f;
+$theme-CeriseRed: #d61f69;
+$theme-PaleSky: #6b7280;
+$theme-Mischka: #d1d5db;
+$theme-AthensGray: #e5e7eb;
+$radius: 0.5rem;
+
+.standupContainer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 1.875rem;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+}
+
+.standupBanner {
+    background-color: $theme-Biscay;
+    width: 83.6875%;
+    height: 12%;
+    color: $theme-white;
+    padding: 1.5rem 0 1.5rem 3.0625rem;
+    border-radius: $radius;
+}
+
+.standupBanner p {
+    font-family: 'Poppins', sans-serif;
+    font-size: 1.25rem;
+    font-weight: 600;
+}
+
+.standupBanner p span {
+    color: $theme-FrenchRose;
+    text-decoration: underline;
+    margin: 0 0.25rem;
+}
+
+.standupUpdateContainer {
+    width: 50.63625%;
```

Comment:
what are these extra decimal places for ?
please refrain yourself from using such kind of styling practices 

`50.6%` more than enough